sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/routing/MurmurHash3_128.java

```diff
@@ -109,12 +109,6 @@ public static UInt128 hash128(byte[] bytes, int length, UInt128 seed) {
         return new UInt128(h1, h2);
     }
 
-
-    private static int rotateLeft32(int n, int numBits) {
```

Comment:
MurMurHash is sort of a util class. Please donot delete these methods even thought they are private as they might be useful in future to do some additional computations as required. Can we just comment those and leave there?

Reply:
addressed in the lastest [commit](https://github.com/Azure/azure-sdk-for-java/pull/9111/commits/b972e8c088bb80842e7fa0633740f49128f1991b).